auth.h

```diff
@@ -111,6 +112,12 @@ struct Authmethod {
 	int	*enabled;
 };
 
+enum KbdintResult {
+	KbdintResultFailure = -1,
+	KbdintResultSuccess,
+	KbdintResultAgain,
+};
+
```

Comment:
This is stylistic, but we generally prefer `#define` to enum in OpenSSH. Defined symbols are easier to grep for

Reply:
Oh, I see... Well I thought they were also more prone to errors, but will do.